terraform/environments/dev/.terraform.lock.hcl

```diff
@@ -0,0 +1,38 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
do we need this file?

Reply:
Yeah you can think of it as package.json, it essentially makes sure every time you do a terraform init you have the same provider version

Reply:
ahh makes sense, ty